### ISSUE TITLE
Simplified Routes Handling in (rootDir)/routes.js

### ISSUE THREAD

**Issue by NikkiDreams** [marko-js/marko-starter#4]
I started using M.S. yesterday. I definitely like where this is going. Especially for building small standalone components that have both font-end and simple back-end concerns. Adding route directories to the root is kind of a pain in the ass. It has merit for sure. But for my project I was hoping to do something like add a routes.js in the root and build from there. It just keeps things simpler. Is that possible with the route-builder or is there a way to do something like the example below?

**routes.js**
```
exports.routes = [
    {
        route:{
            path: '/',
            metadata: {},
            params: {},
            handler: (req, res) => {
                indexTemplate.render({
                    componentName: 'One Time Payments'
                }, res);
            }
        }
    },
    {
        route:{
            path: '/foo',
            metadata: {},
            params: {},
            handler: (req, res) => {
                return re;
            }
        }
    }
];
```

Another thought too is if it were possible to incorporate this in to the project.js file. Something like this would be pretty convenient.

```
const lassoConf = require('./config/lasso.json');
module.exports = require('marko-starter').projectConfig({
    routePathPrefix: '/',
    lassoConfig: lassoConf,
    routes:  [
        {
            route: {
                path: '/',
                metadata: {},
                params: {},
                handler: (req, res) => {
                    indexTemplate.render({
                        componentName: 'foo'
                    }, res);
                }
            }
        }, {
            route: {
                path: '/foo',
                metadata: {},
                params: {},
                handler: (req, res) => {
                    return res;
                }
            }
        }
    ]
});
```

OR even simple like this that could be used in routes.js or project.js

```
routes: {
 {
        "route": "GET /index/:foo",
        "path": "CharityDashboard",
        "metadata": {foo:bar, poo:bucket},
        "params": {},
        "handler": (req,res)=>{
            const indexTemplate = require('./test/test-page/index.marko');
            indexTemplate.render({
                    componentName: 'One Time Payments'
                }, res); 
         }
    }
}
```

**Comment by austinkelleher** [comment]
I agree that something like this should be supported. I think it should be added to the `project.js` file. It looks like right now we are overriding the routes that could be provided in the config and reading from the file system. I will go ahead and fix this. Then you will be able to do:

```js
module.exports = require('marko-starter').projectConfig({
    ...
    routes:  [
        {
                path: '/',
                metadata: {},
                params: {},
                handler: (req, res) => {
                    indexTemplate.render({
                        componentName: 'foo'
                    }, res);
                }
        }, {
                path: '/foo',
                metadata: {},
                params: {},
                handler: (req, res) => {
                    return res;
                }
        }
    ]
});
```

If you want to provide your own `route.js` file to externalize the routes, you could do:

```js
module.exports = require('marko-starter').projectConfig({
    ...
    routes:  require('./routes')
});
```

**Comment by austinkelleher** [comment]
Published `marko-starter@1.1.6` with allowing `routes` to be provided in `projectConfig`.
